src/ofono/provider_ofono.cpp

```diff
@@ -957,6 +977,50 @@ void Bridge::setup_sim(QString const &path)
         setup_stk(modem_path_);
 }
 
+void Bridge::setup_callManager(QString const &path)
+{
+    qDebug() << "Get call properties";
+    auto update = [this](QString const &n, QVariant const &v) {
+        DBG() << "call prop: " << n << "=" << v;
```

Comment:
DBG is compiled in only on debug builds, while qDebug() inlines quite a lot of code